nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,43 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it, *item;
+    PyObject *(*iternext)(PyObject *);
+    int cmp;
+
+    it = PyObject_GetIter(value);
+    if (it == NULL)
+        return NULL;
+    iternext = *Py_TYPE(it)->tp_iternext;
+
+    for (;;) {
+        item = iternext(it);
+        if (item == NULL)
+            break;
+        cmp = PyObject_IsTrue(item);
+        Py_DECREF(item);
+        if (cmp < 0) {
```

Comment:
For cases when an exception is raised, i.e. typically `NULL` returns, the condition is unlikely true. For branch prediction it seems the compilers enjoy a hint about it, and it's also for readability useful, check out the many uses of `unlikely` and sometimes `likely`. If we expect that a branch is practically never taken, we should be able to tell from the source immediately. Therefore `if (unlikely(...)))` is also information.